app/src/main/java/com/peacecorps/pcsa/circle_of_trust/DialogActivity.java

```diff
@@ -0,0 +1,48 @@
+package com.peacecorps.pcsa.circle_of_trust;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.graphics.Rect;
+import android.os.Bundle;
+import android.view.View;
+import android.view.Window;
+import android.view.WindowManager;
+import android.widget.TextView;
+
+import com.peacecorps.pcsa.Constants;
+import com.peacecorps.pcsa.R;
+
+public class DialogActivity extends Activity {
```

Comment:
Rename this class as SMSReportDialogActivity since this is not a common DialogActivity which can be used elsewhere